tox.ini

```diff
@@ -20,7 +23,14 @@ deps =
     pytest61: pytest==6.1.*
     pytest62: pytest==6.2.*
     pytest70: pytest==7.0.*
+    pytest71: pytest==7.1.*
+    pytest72: pytest==7.2.*
+    pytest73: pytest==7.3.*
+    pytest74: pytest==7.4.*
     pytestdev: git+https://github.com/pytest-dev/pytest#egg=pytest
+    # We keep seeing pytest related issues with the latest release and python 3.12, keep using the dev version until it's fixed upstream
```

Comment:
Oh, interesting. Should we cross-link those issues here?

Reply:
Not sure how real those issues are, I would just leave this for now as is and open a reminder that we need to check what our minimum pytest requirement should be for 3.12. (e.g. I don't think it makes sense just yet to add any limits to setup.cfg, would you disagree?)

Reply:
That is fine by me!